Gems/Atom/Tools/ShaderManagementConsole/Code/Source/Document/ShaderManagementConsoleDocument.cpp

```diff
@@ -78,7 +80,7 @@ namespace ShaderManagementConsole
                 variantInfo.m_options[optionDescriptor.GetName()] = ""; // all unset
             }
             variantInfo.m_stableId = m_shaderVariantListSourceData.m_shaderVariants.empty()
-                ? 0
+                ? 1
```

Comment:
Please add a comment or constant to explain why the value needs to be 1 instead of zero